airflow/www/static/js/utils/index.ts

```diff
@@ -190,6 +192,15 @@ const highlightByKeywords = (
   errorKeywords: string[],
   warningKeywords: string[]
 ): string => {
+  const containsLogMarker = logMarkers.some((marker) =>
+    parsedLine.includes(marker)
```

Comment:
As above: Should be a regex match, not a exact text match

Reply:
Updated to regex match along with tests. Thanks.